src/code.cloudfoundry.org/loggregator/doppler/internal/server/v1/doppler_server.go

```diff
@@ -170,24 +172,24 @@ func (m *DopplerServer) sendData(req *plumbing.SubscriptionRequest, sender sende
 			return err
 		}
 
-		// metric-documentation-v2: (loggregator.doppler.egress) Number of
```

Comment:
Why did you move the doc comment? I believe @enocom put these at invocation site not where they were constructed.